index.js

```diff
@@ -1490,6 +1490,38 @@
         return either.isLeft ? l(either.value) : r(either.value);
       });
 
+  //# lefts :: [Either a b] -> [a]
+  //.
+  //. Takes a list of Eithers and returns a list containing each Left's value.
+  //.
+  //. ```javascript
+  //. > S.lefts([S.Right(20), S.Left('foo'), S.Right(10), S.Left('bar')])
+  //. ['foo', 'bar']
+  //. ```
+  S.lefts =
+  def('lefts',
+      {},
+      [$.Array($Either(a, b)), $.Array(a)],
+      R.chain(function(either) {
```

Comment:
In both of these cases, we could alternatively filter -> map to achieve the same ends.